mappings/net/minecraft/entity/ai/goal/MoveToVillageCenterGoal.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS akj net/minecraft/entity/ai/goal/MoveToVillageCenterGoal
+	FIELD a owner Laib;
+	FIELD b searchRange I
+	FIELD c inversedChance I
```

Comment:
I don't like this name very much

Reply:
So `reciprocalChance`?

Reply:
Works for me; can't think of a better one